llvm/test/CodeGen/RISCV/attributes.ll

```diff
@@ -520,3 +520,8 @@ define i8 @atomic_load_i8_seq_cst(ptr %a) nounwind {
 ; A6S: .attribute 14, 2
 ; A6C: .attribute 14, 1
 }
+
```

Comment:
No CHECKs for this test or I miss something here?

Reply:
Sorry, I'm a little stuck on how to infer the attribute. This function will get a test once I have that figured out.